src/controllers/expense.controller.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-console */
+'use strict';
+
+const expenseService = require('../services/expense.service');
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  let filteredExpenses = expenseService.getAll();
+
+  if (userId) {
+    filteredExpenses = expenseService.getAllByUserId(+userId, filteredExpenses);
+  }
+
+  if (categories) {
+    filteredExpenses = expenseService
+      .getAllByCategory(categories, filteredExpenses);
+  }
+
+  if (from) {
+    filteredExpenses = expenseService.getAllByDateFrom(from, filteredExpenses);
+  }
+
+  if (to) {
+    filteredExpenses = expenseService.getAllByDateTo(to, filteredExpenses);
+  }
+
+  res.statusCode = 200;
+  res.send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const expenseId = +req.url.slice(1);
```

Comment:
```suggestion
  const expenseId = Number(req.params.id);
```

The same in other places in code
Use Number it's easier for reading